module2/README.md

```diff
@@ -0,0 +1,97 @@
+
+h) How can you view your App in Light and Dark Modes simultaneously?
+
+Setup multiple preview views with .preferredColorScheme set to .light and .dark 
+
+i) Below is an image of the Canvas from Xcode. The Canvas is in selectable mode. Can you explain why the red background does not cover the entire button area?
+
+It is because a padding modifier is being appled to the view after the red background. The red-backbground modifier returns a new view which is the same size as the original view, but with a red-background. Applying a padding modifier after this returns a new view with white padding around the view with the red background.
+
+j) Modifier padding(10) adds padding to the view's top, bottom, left, and right sides. How could a padding of 10 be added to only the left and right sides of the view? The answer for this question should be a short section of code.
+
+Use the modifier .padding(.horizontal, 10)
+
+k) Provide two reasons why you would want to extract views.
+
+One - to re-use the views as subviews in multiple parent views
+Two - to keep the code for views from being too long and hard to understand
+
+l) How can you determine, in code, if the device is in Portrait or Landscape mode?
+
+You can approximate this with the environment variables verticalSizeClass and horizontalSizeClass. If vertical is compact and horizontal is regular, then you are in Landscape. If vertical is regular and horizontal is compact, then you are in Portrait. This would work for phone style devices. 
+
+m) What is a literal value?
+
+A literal value is an inline expression of a value that is implicitly initialized at instantiation time. An example would be initializing an array: 
+
+var myArray = [1, 2, 3]
+
+Where the "[1, 2, 3]" is a literal value.
+
+o) What are the safe areas?
+
+Safe areas are the edges of the screen UI that might be used for the notch or other OS related UI like a grabbable scroll nub or bar.
+
+p) This line of code was in the lesson on animation. Can you state in English what the line means?
+
+.frame(width: wideShapes ? 200 : 100)
+
+If the boolean variable "wideShapes" is to, apply the frame() view modifier to the view above it with a width of 200. If "wideShapes" is false, apply the frame() view modifier with a width of 100.
+
+q) Describe the two transitions you were introduced to in this week’s lesson.
+
+.transition(.scale) - which transitions the size of a view as its states change
+.transition(.opacity) - which fades a view in and out (default)
+
+r) In Bullseye, the Game struct is what type of object?
+
+It performs the role of "model" in the app architecture. It is a struct.
+
+s) What are SFSymbols?
+
+SFSymbols are a set if icons Apple provides which you can download and incorporate into your app.
```

Comment:
They are designed to look correct when displayed with Apples San Francisco font text.